java-tools/OsmAndServer/src/main/java/net/osmand/server/index/dao/AbstractDAO.java

```diff
@@ -0,0 +1,51 @@
+package net.osmand.server.index.dao;
+
+import net.osmand.server.index.reader.DirectoryReader;
+import net.osmand.server.index.type.Type;
+import net.osmand.server.index.type.TypeFactory;
+
+import java.io.IOException;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+
+public abstract class AbstractDAO {
+
+    private static final Path BASE_PATH = Paths.get("/var/www-download/");
```

Comment:
This definitely should be parameterizable and better to use application.properties